.github/workflows/e2e-test.yaml

```diff
@@ -0,0 +1,32 @@
+name: 'e2e-test: install chart on kind'
+on: pull_request
+
+jobs:
+  test-install:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.3.0  
+
+      - name: install chart
+        run: |
+          sed -i 's/\[\[ .Version \]\]/0.0.0/' helm/prometheus-operator-app/Chart.yaml
```

Comment:
Should we not deploy the CRD also?

Reply:
this is for testing! 
This dependency has the CRDs packaged

Reply:
ok :)

Reply:
Does that mean we do not the the prometheus-operator-crd app anymore?

Reply:
We want to manage the CRDs via a standalone app,
because of CRDs upgrade, 
The `crds` folder will ignore deploying CRDs if they exists.
Hey @TheoBrigitte

Reply:
Sure, but in that case, should we not deploy the prometheus-operator-crd app as a dependency to this app? and also, I would prefer that the crd are disabled by default in the values

Reply:
Because this will generate ownership issues

Reply:
We can't disable them via values, 
they are disabled if the CRDs already exists

Reply:
chart dependency does not work for CRDs
having an App CR for would not work either

I follow Quentin here and I would prefer that we remove CRDs from the chart, otherwise we will run into issues.

Reply:
we can do that by setting `createCustomResource: false` right?

Reply:
createCustomResource does not exist upstream. It's an old value :(